plugins/gatsby-source-data/main.js

```diff
@@ -37,8 +36,8 @@ const dataExists = name => {
 const getLiveSessionsFromSheets = async apiKey => {
   const sheets = new Sheets(apiKey);
   const values = await sheets.getLiveSessions();
-  values.shift() // Get rid of the first row which is just the title
-  const keys = values.shift();
+  values.shift().shift() // Get rid of the first row which is just the title
```

Comment:
_nitpick_

1. In the event `len(values) < 2`, the above line will fail, use `slice` instead.
2. Rename it as `liveSessions`.
```
let liveSessions = await sheets.getLiveSessions();
liveSessions = values.slice(2) // Get rid of first two rows
```